doc/man1/flux-jobs.adoc

```diff
@@ -0,0 +1,100 @@
+// flux-help-include: true
+FLUX-JOBS(1)
+==============
```

Comment:
I found the  formatted man page seems to be indented far to the right.  It seems to be fixed by making the length of the `====` underline match the length of the title.  (This is with asciidoctor as formatter)